deployment/provision_vagrant_vm.sh

```diff
@@ -37,11 +37,10 @@ service redis-server restart
 apt-get -q install -y apache2 apache2-dev libapache2-mod-wsgi-py3
 
 # With docker for mac, root will own the mount point (uid=0). chmod does not touch the host file system in these cases.
-OWNER=$(stat -c %U "$MOUNTPOINT/evap")
-if [ "$OWNER" == "root" ]; then chown -R 1042 "$MOUNTPOINT"; fi
+if [ $(stat -c "%u" "$MOUNTPOINT/evap") == "root" ]; then chown -R 1042 "$MOUNTPOINT"; fi
 
 # make user, create home folder, set uid to the same set in the Vagrantfile (required for becoming the synced folder owner), set default shell to bash
-useradd -m -u $(stat -c "%u" "$MOUNTPOINT/evap") -s /bin/bash evap
+useradd -m -u 1042 -s /bin/bash evap
```

Comment:
breaks ubuntu docker setup for me. With native docker, the folder is owned by the user with id 1000. For me, with your setup:
```
$ stat -c "%u" "/evap"
1000  # vagrant mount did not use the `mount_options` we specified
$ sudo cat /etc/passwd
-> root has user id 0  # as expected
```
but, due to this change, the user `evap` doesn't own `/evap` anymore, and thus all file accesses to that folder fail.


I think the `$(stat -c "%u" "$MOUNTPOINT/evap")` here was added because vagrant doesn't correctly apply the mount options of uid and gid that we provide in the Vagrantfile. In any case, whether this folder is mounted for uid 1000 or 1042 or any other value, we want the evap user to have that user id. Going back to hardcoding this value doesn't seem to be an option to be, considering this background.